Binance.Net/Objects/Spot/WalletData/BinanceTradingStatus.cs

```diff
@@ -9,11 +9,13 @@ namespace Binance.Net.Objects.Spot.WalletData
 {
     internal class BinanceTradingStatusWrapper
     {
-        public bool Success { get; set; }
-        [JsonProperty("msg")]
-        [JsonOptionalProperty]
-        public string? Message { get; set; }
-        public BinanceTradingStatus? Status { get; set; }
+        /// <summary>
+        /// Time account status was updated
+        /// </summary>
+        [JsonConverter(typeof(TimestampConverter))]
+        public DateTime UpdateTime { get; set; }
```

Comment:
I don't see `UpdateTime` neither in docs not in current API output. It's present only inside `Data` section